src/components/team.js

```diff
@@ -1,11 +1,77 @@
 import React from "react"
+import { graphql, useStaticQuery } from "gatsby"
 import "./team.scss"
 import { useWindowSize } from "./hooks"
 
 import TeamMember from "./teamMember"
 
 export default function Team({ team }) {
```

Comment:
Are we still using this prop, or can it be removed?

Reply:
Good catch, that one is no longer needed. I'll get that prop out of there.